src/star/star_solo/argument_groups_solo.yaml

```diff
@@ -0,0 +1,230 @@
+      type of UMI deduplication (collapsing) algorithm
+
+      - 1MM_All                     ... all UMIs with 1 mismatch distance to each other are collapsed (i.e. counted once).
+      - 1MM_Directional_UMItools    ... follows the "directional" method from the UMI-tools by Smith, Heger and Sudbery (Genome Research 2017).
+      - 1MM_Directional             ... same as 1MM_Directional_UMItools, but with more stringent criteria for duplicate UMIs
+      - Exact                       ... only exactly matching UMIs are collapsed.
+      - NoDedup                     ... no deduplication of UMIs, count all reads.
+      - 1MM_CR                      ... CellRanger2-4 algorithm for 1MM UMI collapsing.
+    example: 1MM_All
+    multiple: yes
+    multiple_sep: ;
+  - name: --soloUMIfiltering
+    type: string
+    description: |-
+      type of UMI filtering (for reads uniquely mapping to genes)
+
+      - -                  ... basic filtering: remove UMIs with N and homopolymers (similar to CellRanger 2.2.0).
+      - MultiGeneUMI       ... basic + remove lower-count UMIs that map to more than one gene.
+      - MultiGeneUMI_All   ... basic + remove all UMIs that map to more than one gene.
+      - MultiGeneUMI_CR    ... basic + remove lower-count UMIs that map to more than one gene, matching CellRanger > 3.0.0 .
+      Only works with --soloUMIdedup 1MM_CR
+    multiple: yes
+    multiple_sep: ;
+  - name: --soloOutFileNames
+    type: string
+    description: |-
+      file names for STARsolo output:
+
+      file_name_prefix   gene_names   barcode_sequences   cell_feature_count_matrix
+    example:
+    - Solo.out/
+    - features.tsv
+    - barcodes.tsv
+    - matrix.mtx
+    multiple: yes
+    multiple_sep: ;
+  - name: --soloCellFilter
+    type: string
+    description: |-
+      cell filtering type and parameters
+
+      - None            ... do not output filtered cells
+      - TopCells        ... only report top cells by UMI count, followed by the exact number of cells
+      - CellRanger2.2   ... simple filtering of CellRanger 2.2.
+      Can be followed by numbers: number of expected cells, robust maximum percentile for UMI count, maximum to minimum ratio for UMI count
+      The harcoded values are from CellRanger: nExpectedCells=3000;  maxPercentile=0.99;  maxMinRatio=10
+      - EmptyDrops_CR   ... EmptyDrops filtering in CellRanger flavor. Please cite the original EmptyDrops paper: A.T.L Lun et al, Genome Biology, 20, 63 (2019): https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1662-y
+      Can be followed by 10 numeric parameters:  nExpectedCells   maxPercentile   maxMinRatio   indMin   indMax   umiMin   umiMinFracMedian   candMaxN   FDR   simN
+      The harcoded values are from CellRanger:             3000            0.99            10    45000    90000      500               0.01      20000  0.01  10000
+    example:
+    - CellRanger2.2
+    - '3000'
+    - '0.99'
+    - '10'
+    multiple: yes
```

Comment:
```suggestion
    multiple: true
```

Reply:
you didn't really have to suggest every single `multiple: yes` to `multiple: true` :P